enatega-multivendor-app/src/context/User.js

```
@@ -82,7 +82,7 @@ export const UserProvider = props => {
     console.log('error context user', error.message)
   }
   async function onCompleted(data) {
-    const { _id: userId, name, email, phone } = data.profile
+    const { _id: userId, name, email, phone } = data?.profile
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)